src/image/image.js

```diff
@@ -341,4 +341,27 @@ p5.prototype._makeFrame = function(filename, extension, _cnv) {
   frames.push(thisFrame);
 };
 
+/**
+ * Returns the current canvas as a loaded p5.Image. This allows an assignment
+ * or immediate use in the users sketch as a canvas screenshot.
+ *
+ * @method getCanvasImage
+ * @param none
+ *
+ * @example
+ * // This example shows how a user can directly create a screenshot of his
+ * // canvas and have it saved for future use or use immediately
+ * draw() {
```

Comment:
it should have been `function draw()`? Also have a look at the other functions and how they format the documentation and examples, you are missing some HTML tags I believe